pkg/sqlcache/db/client.go

```diff
@@ -21,11 +24,13 @@ import (
 
 	// needed for drivers
 	_ "modernc.org/sqlite"
+	sqlite "modernc.org/sqlite"
 )
 
 const (
 	// InformerObjectCacheDBPath is where SQLite's object database file will be stored relative to process running steve
```

Comment:
We should move this comment to stay with the the `const` it is describing

Reply:
I'm referring to the `InformerObjectCacheDBPath` constant. We moved it below `InformerObjectCacheDBPathRoot` so in the docs we'll have a description for `InformerObjectCacheDBPath` but on the wrong const `InformerObjectCacheDBPathRoot`

Reply:
Related change: the `cleanTempFiles` function in `pkg/sqlcache/informer/listoption_indexer_test.go` to remove the 3 sqlite files after the test finishes.